### PR TITLE
more detailed privacy considerations

### PR DESCRIPTION
**Pull request by chaals**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -885,17 +885,15 @@ <h2>
...
-      </section>
-
-      <section>
-        <h2>Access to aspects of a user’s local computing environment</h2>
-        <p>
-          The screen orientation type and angle of the device can be accessed with the API specified in this document, and can be a potential fingerprinting vector.
+          The screen orientation type and angle of the device can be accessed with the API specified in this document,
+          and can be a potential fingerprinting vector. However using this information is very difficult compared to other fingerprinting information. 
+          Aggregating information derived from this API is not very helpful in fingerprinting. 
+          In particular the Device Orientation API provides a far more detailed version of the same information.
```
Comment (xfq) [inline index.html]: By "Device Orientation API", do you mean the [DeviceOrientation Event Specification](https://w3c.github.io/deviceorientation/spec-source-orientation.html), or the [motion sensor specs](https://w3c.github.io/motion-sensors/)?

Comment (chaals) [reply index.html]: I meant the event spec, but actually it is true of both.

**Comment by xfq** [comment]
I updated the text a bit. Could you please check if it's good enough?

**Comment by chaals** [comment]
Looks good enough  to me - thank you...

**Comment by xfq** [comment]
Merging.

**Pull request merged**